libs/deps/merch-card.js

```diff
@@ -1,4 +1,4 @@
-// branch: main commit: fbbefe1b71216b73b1e41d50997927fbc7340473 Tue, 25 Jun 2024 15:33:55 GMT
+// branch: rosahu/MWPW-147172 commit: 9a187c62466b757074fb79bf342cfdaa54212316 Tue, 02 Jul 2024 14:56:12 GMT
```

Comment:
@rohitsahu please only use artifacts built from main branch (can be ok for temp review, but then add do not merge label)

Reply:
sure, waiting for mas PR merge to main, then will update the merch-card.js file

Reply:
updated now